### DISCUSSION TITLE
The "," for decimals

### DISCUSSION THREAD

**Discussion by stepsolar** [ExperienceLovelace/ha-floorplan#308] [Q&A] [answered]
Hi, why for decimals there is the "."  instead of the ","?  It would be possible to change the "."  with the ","?  Thank you

**Comment by exetico** [comment, marked as answer]
Decimals? You're not telling us where.

If you think within HA entities, what's just the standard - create a custom entity (could be as a custom template sensor), if you need something specific. 

If it's in a case where you're trying to expose the entity state (as 14.4 in the screenshot), replace the dot with a comma in your JS code 🤔? If its a number, you can use the toLocalString() function. 

Again. Remember the content stepsolar, thanks.

Btw: There's plenty of ressources online, so I'll recommend you to search a bit for the answers, before creating a thread here. 

There's some example ressources:
https://www.fwait.com/how-to-replace-dot-with-comma-in-javascript/

https://www.javatpoint.com/javascript-format-numbers-with-commas

![Screenshot_20221227_191959_Home Assistant](https://user-images.githubusercontent.com/3549445/209707048-84a3cffa-897e-4968-958c-87dfa39a29fa.jpg)
